packages/backend-for-frontend/src/utilities/errorMappers.ts

```diff
@@ -80,6 +80,7 @@ export const sessionTokenErrorMapper = (error: ApiError<ErrorCodes>): number =>
     .with("tokenVerificationFailed", () => HTTP_STATUS_UNAUTHORIZED)
     .with("tenantLoginNotAllowed", () => HTTP_STATUS_FORBIDDEN)
     .with("tooManyRequestsError", () => HTTP_STATUS_TOO_MANY_REQUESTS)
+    .with("genericError", () => HTTP_STATUS_BAD_REQUEST)
```

Comment:
a `genericError` should always be an internal server error because it has the meaning "we don't know what happened".
If you need to intercept a specific condition it's better to create a new error.
In this case something like `MissingUserRolesInIdentityToken`